setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, roots):
 
 setup(
     name='html-xblock',
-    version='0.1.3',
+    version='0.2.0',
```

Comment:
This is a breaking change, so we should bump MAJOR. [Ref](https://semver.org/).

Reply:
@Agrendalath, I mentioned this in one of my review comments. But I wasn't sure if we could call this XBlock 1.0, as that version has some implicit expectations about the quality, maturity, stability etc.

@taniwha, can you bump the major version in `setup.py`?